pkg/testutil/testutil.go

```diff
@@ -33,6 +33,7 @@ func WaitUntil(c *check.C, f CheckFunc) {
 	c.Log("wait start")
 	for i := 0; i < waitMaxRetry; i++ {
 		if f(c) {
+			c.Succeed()
```

Comment:
Why do you want to add this?

Reply:
`Check` will mark the test failed if no pass although it will not exit immediately.